httpcache.go

```diff
@@ -278,6 +298,19 @@ func parseCaddyfileGlobalOption(h *caddyfile.Dispenser, _ interface{}) (interfac
 					}
 				}
 				cfg.API = apiConfiguration
+			case "badger":
```

Comment:
This should also get documented in the Caddyfile

Or better, we should have the full Caddyfile syntax in the README.